runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/ParserExec/PredicatedIfIfElse.stg

```diff
@@ -25,7 +25,7 @@ grammar(grammarName) ::= <<
 grammar <grammarName>;
 s : stmt EOF ;
 stmt : ifStmt | ID;
-ifStmt : 'if' ID stmt ('else' stmt | { <LANotEquals("1", "ELSE")> }?);
+ifStmt : 'if' ID stmt ('else' stmt | { <LANotEquals("1", {<grammarName>Parser.ELSE})> }?);
```

Comment:
:exclamation: A test needs to be added to ensure `ELSE` (unqualified) remains a valid reference in code generated by the Java target.